react/features/invite/functions.js

```diff
@@ -389,6 +390,20 @@ export function searchDirectory( // eslint-disable-line max-params
             });
 }
 
+/**
+ * Returns true if the current deployment supports dial number retreival.
+ *
+ * @param {Object | Function} stateful - An instance that can be resolved into the Redux state.
+ * @returns {boolean}
+ */
+export function canGetDialInfo(stateful: Object | Function): boolean {
+    const { dialInConfCodeUrl, dialInNumbersUrl, hosts }
+                = toState(stateful)['features/base/config'];
+    const mucURL = hosts && hosts.muc;
```

Comment:
Not sure if checking the MUC URL is too much, without it nothing will work.

Reply:
Right, but this is not invoked, it was just a remainder of a refactor so I removed it. Thanks!